android/build.sh

```diff
@@ -0,0 +1,87 @@
+# defining params
+# TODO: should only build one platform by default - will change when ADO pipeline is set up
```

Comment:
>should only build one platform by default [](http://example.com/codeflow?start=8&length=41)

The script can be parameterized to take a list of platforms as input .. In ADO pipelines, when using input matrix, the script will be called with one config at a time.

Reply:
Yes

Reply:
I'm adding this in another PR with the ADO files.